### PR TITLE
create a PR template for limited-partnership-api

### PR DESCRIPTION
**Pull request by HatiDev**
https://companieshouse.atlassian.net/browse/LP-169

create a PR template for limited-partnership-api

### CODE REVIEW

**Review by markpit — APPROVED** [review]

**Review by lduranteau — APPROVED** [review]

File: .github/PULL_REQUEST-TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+
+### Change description
+
+
+
+### Work checklist
+
+- [ ] Tests added where applicable
```
Comment (makhtar-ch) [inline .github/PULL_REQUEST-TEMPLATE.md:11]: Is a checklist for UI changes on an API really necessary? And the gobbledygook merge instructions at the bottom? I noticed a lot of APIs use something like https://github.com/companieshouse/acsp-api/blob/main/.github/PULL_REQUEST_TEMPLATE.md . Wondering if its worth tidying up this one?

Comment (HatiDev) [reply .github/PULL_REQUEST-TEMPLATE.md]: Changed the template to match @makhtar-ch request

Comment (lduranteau) [reply .github/PULL_REQUEST-TEMPLATE.md]: now the test line is missing

- [ ] Tests added where applicable

Comment (HatiDev) [reply .github/PULL_REQUEST-TEMPLATE.md]: line added

**Review by lduranteau — APPROVED** [review]

**Review by makhtar-ch — APPROVED** [review]

File: .github/PULL_REQUEST-TEMPLATE.md (hunk 1 above)
Comment (dlloyd1-ch) [reply .github/PULL_REQUEST-TEMPLATE.md]: The test line is in the PR checklist section. Now we have two.

**Pull request merged**